src/psyclone/psyir/nodes/omp_directives.py

```diff
@@ -1406,25 +1406,6 @@ def end_string(self):
         # pylint: disable=no-self-use
         return "omp end target"
 
-    def validate_global_constraints(self):
-        '''
-        Perform validation checks that can only be done at code-generation
-        time.
-
-        TODO #1837. This should be expanded to all intrinsics not supported
-        on GPUs. But it may be implementation-dependent!
-
-        :raises GenerationError: if this OMPTargetDirective contains \
-            CodeBlocks.
-        '''
-        super().validate_global_constraints()
-
-        cbs = self.walk(CodeBlock)
```

Comment:
I've realised pylint complains about an unused import of CodeBlock in this file too.

Reply:
Removed